tests/unit_tests/test_tethys_services/test_backends/test_hydroshare.py

```diff
@@ -54,17 +54,12 @@ def test_extra_data(self):
             scope="scope",
         )
 
-        mock_details = dict(
-            name="",
-            alias="",
-        )
-
         hydro_share_auth2_obj = HydroShareOAuth2()
 
         hydro_share_auth2_obj.set_expires_in_to = 100
 
         ret = hydro_share_auth2_obj.extra_data(
-            "user1", "0001-009", mock_response, mock_details
```

Comment:
I'd prefer @sdc50's solution here. @romer8 can you change it to this?

Reply:
I have implemented this solution in the latest commit: https://github.com/romer8/tethys/commit/009a83e5efa8a9af51ed334c7bf20a4559f43093

Reply:
I have also merge the main branch for the latest change